sql/nondesktop_exact_mau28_by_dimensions_v1.sql

```diff
@@ -0,0 +1,49 @@
+WITH
+  inactive_days AS (
+    SELECT
+      *,
+      DATE_DIFF(submission_date, date_last_seen, DAY) AS _inactive_days,
+      DATE_DIFF(submission_date, date_last_seen_in_tier1_country, DAY) AS _inactive_days_tier1
+    FROM
+      core_clients_last_seen_v1
+  )
+
+SELECT
+  submission_date,
+  CURRENT_DATETIME() AS generated_time,
+  COUNTIF(_inactive_days < 28) AS mau,
+  COUNTIF(_inactive_days < 7) AS wau,
+  COUNTIF(_inactive_days < 1) AS dau,
+  -- You can recover MAU for all clients in tier 1 countries via a WHERE clause
+  -- and summing mau, but this considers only the country associated with the
+  -- latest observation of the client; "Tier 1 grouped" MAU is an alternative
+  -- that includes all clients seen in a tier 1 country in the 28 day window,
+  -- even if their latest ping came from an excluded country.
+  COUNTIF(_inactive_days_tier1 < 28) AS tier1_grouped_mau,
```

Comment:
I initially used `exact_mau28` to be very self-documenting. No question about whether it's exact or approximate, no question about whether it's 28-day, 30-day, or calendar months.

As the table expands to cover more use cases, it may indeed be time to pick a more descriptive / generalized name. For now, I am fine with leaving the table name overly specific with various "bonus" columns, and leave a naming strategy to a follow-up (something something daily metrics?).